### ISSUE TITLE
Words like "yes" and "no" do not appear

### ISSUE THREAD

**Issue by TheRedstoneScientist** [mlomb/chat-analytics#109]
When attempting to search for the words "yes" or "no," these terms do not show up despite the dataset showing otherwise.

I also tried using regex to do
^yes$
^yes
yes$
etc.

**Comment by mlomb** [comment]
Hey! Yeah "yes" and "no" are considered skipwords as you can see in https://github.com/mlomb/chat-analytics/blob/main/assets/data/text/stopwords-iso.json. This is meant to skip words like "the" or "a".

Maybe we can make an exception for those

**Comment by TheRedstoneScientist** [comment]
I see. I was curious to analyze "yes" versus "yeah" and how often we said no to each other over time
  

  

  
  
Ilan Komrad
  
AKA TheRedstoneScientist (TRS)
  
Director: Creative  &  Operations
  
Everbloom Games
  
www.everbloomgames.com
  

  

        
  
  
>

**Comment by Cedric-Boucher** [comment]
yeah, it could make sense to hide stopwords like "yes" and "no" from the word rankings, as is the case currently, but show the word if it is searched for
